src/layman/layer/rest_workspace_layer_style_external_image.py

```diff
@@ -28,5 +31,9 @@ def after_request(response):
 @bp.route(f"/{LAYER_REST_PATH_NAME}/<layername>/style/external_images/<filename>", methods=['GET'])
 def get(workspace, layername, filename):
     app.logger.info(f"GET Style External Image, actor={g.user}, workspace={workspace}, layername={layername}, filename={filename}")
+    image_path = input_style.get_external_image_path(workspace, layername, filename)
+    if image_path:
+        extension = os.path.splitext(image_path)[1][1:]
+        return send_file(image_path, mimetype=f'image/{extension}')
```

Comment:
For *.svg, MIME type is `'image/svg+xml`
Try to use https://docs.python.org/3/library/mimetypes.html